### PR TITLE
ACS-4369 Support for path features.

### PR DESCRIPTION
**Pull request by tpage-alfresco**
PARENT, PRIMARYPARENT and ANCESTOR are now all supported. There is no support for secondary parent-child relationships yet.

Add examples of how to rewrite certain unsupported queries to work around issues.

### CODE REVIEW

File: search-enterprise/latest/using/unsupported.md [hunk 1]
```diff
@@ -58,18 +55,25 @@ The following features, which were supported with Search and Insight Engine 2.x
...
+Alfresco Search Enterprise has a focus on the most commonly used features and often it is possible to work around unsupported features. Here are a few examples of how to use a different field for queries:
+
+| Old Query                                      | Replacement Query                             |
+| ---------------------------------------------- | --------------------------------------------- |
+| QNAME:'comment'                                | TYPE:'fm:post'                                |
+| PNAME:'0/wiki'                                 | PATH:'//cm:wiki/*'                            |
+| NPATH:'2/Company Home/Sites/swsdp'             | PATH: '/app:company_home/st:sites/cm:swsdp/*' |
+| ANAME:'0/cdefb3a9-8f55-4771-a9e3-06fa370250f6' | PARENT:'cdefb3a9-8f55-4771-a9e3-06fa370250f6' |
+
+However secondary paths and secondary parents are not yet supported, so there may still be some differences if these are in use.
```
Comment (mjstanford) [inline search-enterprise/latest/using/unsupported.md:76]: I would rephrase as "However secondary paths and secondary parents are not supported at this time, so there may still be some differences if these are in use."

Using "not yet" implies that these will be supported at some point, and we have not committed to that (yet).

Comment (tpage-alfresco) [reply search-enterprise/latest/using/unsupported.md]: Done :+1:

**Comment by tpage-alfresco** [comment]
@anxumalo This PR is now ready to go live - it's valid for ACS 7.4. I will split the bits targeting 23.1 into a separate PR and raise a DOCS ticket to merge it later.

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM - made some minor adjustments

**Review by oliveralfresco2018 — DISMISSED** [review]
LGTM - made some minor adjustments

**Review by mjstanford — DISMISSED** [review]

**Review by oliveralfresco2018 — APPROVED** [review]
LGTM
